packages/react-components/src/components/Tooltip/Simple.tsx

```diff
@@ -0,0 +1,5 @@
+import * as React from 'react';
+
+export const Simple: React.FC<{ text: string }> = ({ text }) => {
```

Comment:
what is the purpose of this component? 🤔  why should I use it instead of `div` element?

Reply:
The idea behind this is that we provide a closed set of components that satisfy the needs of AA. We wanted to avoid situations when the content of the tooltip does not follow the style guidelines provided by DS.

Reply:
cool, sounds reasonable 👍